include/knowhere/comp/knowhere_check.h

```diff
@@ -43,92 +43,6 @@ SupportMmapIndexTypeCheck(const std::string& index_name) {
     }
 }
 
-inline bool
-CheckBooleanInJson(const knowhere::Json& json, std::string key) {
```

Comment:
Should we move other static check in this file to `include/knowhere/index/index_static.h`?

Reply:
These simple and straightforward checks will be moved into a bitset-based feature structure. Only complex and dynamic static function will move to IndexStaticFaced.